### PR TITLE
Build docs

### PR DESCRIPTION
**Pull request by tznind**
Hi @En3Tho, this is my personal account.  I've been getting ready to tackle #9  but first I needed to work out how to build/test the module.

These are the instructions I created.  Let me know if there is a far easier way to do that and I can update them.

### CODE REVIEW

**Comment by En3Tho** [comment]
@tznind Nice! Seems like a great end to end tutorial. I will take a closer look a little bit later but for now I can say that the easiest way just to test the logic/UI is to use PoshRedisViewerHost project as it's just a console app. 

I guess the doc can include 2 sections: one with easy way (host) and the other the hard way (module).

 Usually module project is needed only for publish or when the command itself changes. Any other time host project is okay. 

You can even use dotnet watch with it. It will always recompile but it's quite fast at doing so.

**Comment by tznind** [comment]
Ah I hadn't realized.  It wasn't set as my startup project in visual studio so struggled a little.

Thanks.

I have updated this PR docs to indicate the two ways of testing.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,66 @@ or
...
+cd PoshRedisViewer
+dotnet publish -o build -r win-x64
+```
+2. Copy the psd1 file into the build directory
+```
+cp .\PoshRedisViewerModule\PoshRedisViewerModule.psd1 .\build\
+```
+3. Start a new powershell instance (prevents stale modules sitting around etc)
+```
+pwsh
```
Comment (En3Tho) [inline README.md]: I think calling `pwsh –noprofile` here is good to be sure no modules are loaded by default

File: PoshRedisViewer/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+build/
```
Comment (En3Tho) [inline PoshRedisViewer/.gitignore]: I personally lean to `.artifacts`. Just to be more consistent with https://github.com/dotnet/core/issues/8135#issuecomment-1483357684

But this is just gitignore  and if you're accustomized to "build" then it's okay

**Review by En3Tho — COMMENTED** [review]
Just a small suggestion. Otherwise everything is perfect. Thanks!

File: README.md (hunk 1 above)
Comment (tznind) [reply README.md]: Cool, thats a handy flag to know about.  I've added it to build instructions

File: PoshRedisViewer/.gitignore (hunk 2 above)
Comment (tznind) [reply PoshRedisViewer/.gitignore]: I have changed from `-o build` to `-o .artifacts` and updated subsequent steps and git ignore file.

**Comment by En3Tho** [comment]
Thanks!

**Pull request merged**